lib/services/event_message_sender.dart

```diff
@@ -10,8 +10,22 @@ class EventMessageSender {
 
     Message message = Message(
         type: MessageType.startEvent,
-        receiverType: MessageReceiverType.all
-    );
+        receiverType: MessageReceiverType.all,
+        timestamp: timestamp);
+
+    newMessage.set(message.toJson());
+  }
+
+  static void endEvent(String eventId) {
```

Comment:
maybe adding a verb here would be more readable idk, sendEndEventMessage

Reply:
end is a verb